client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -129,42 +165,35 @@ class CancelPurchase extends Component {
 		);
 	}
 
-	renderFooterText = () => {
-		const { purchase } = this.props;
-		const { refundText, expiryDate, totalRefundText } = purchase;
-
-		if ( hasAmountAvailableToRefund( purchase ) ) {
-			if ( this.state.cancelBundledDomain && this.props.includedDomainPurchase ) {
-				return this.props.translate( '%(refundText)s to be refunded', {
-					args: { refundText: totalRefundText },
-					context: 'refundText is of the form "[currency-symbol][amount]" i.e. "$20"',
-				} );
-			}
-			return this.props.translate( '%(refundText)s to be refunded', {
-				args: { refundText },
-				context: 'refundText is of the form "[currency-symbol][amount]" i.e. "$20"',
-			} );
-		}
-
-		const expirationDate = this.props.moment( expiryDate ).format( 'LL' );
-
-		if ( isDomainRegistration( purchase ) ) {
-			return this.props.translate(
-				'After you confirm this change, the domain will be removed on %(expirationDate)s',
-				{
-					args: { expirationDate },
-				}
-			);
-		}
-
+	supportLink = () => {
 		return this.props.translate(
-			'After you confirm this change, the subscription will be removed on %(expirationDate)s',
+			'Have a question? {{contactLink}}Ask a Happiness Engineer!{{/contactLink}}',
 			{
-				args: { expirationDate },
+				components: {
+					contactLink: <a href={ CALYPSO_CONTACT } />,
+				},
 			}
 		);
 	};
 
+	keepButtonCTA = ( locale ) => {
+		let cta;
+
+		if ( isJetpackProduct ) {
```

Comment:
Since `isJetpackProduct` is a function imported from `@automattic/calypso-products`, this will always be true. Did you mean to check `this.props.isJetpackPurchase`?

Reply:
Yes I meant `this.props.isJetpackPurchase`, I fixed this issue